activestorage/lib/active_storage.rb

```diff
@@ -68,6 +68,8 @@ module ActiveStorage
   mattr_accessor :replace_on_assign_to_many, default: false
   mattr_accessor :track_variants, default: false
 
+  mattr_accessor :video_preview_arguments, default: ["-y", "-vframes", "1", "-f", "image2", "-"]
```

Comment:
Sorry, I realize now that I commented on the backport PR but pasting that here for context.

Does an array make sense here? If i'm trying to override this behavior I would expect to pass a single string as the arguments.

Reply:
Yes, I think the consensus is to go with a string

Reply:
Correct.